### PR TITLE
Add note to link to v1 README

### PR DESCRIPTION
**Pull request by jzjiang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # [Shopify](https://www.shopify.com) JavaScript Buy SDK
 [![Circle CI](https://circleci.com/gh/Shopify/js-buy-sdk.png?circle-token=3be0ebe6fbb4841442b86678696947bd4b5456d7)](https://circleci.com/gh/Shopify/js-buy-sdk)
 
+**Note**: v1 of the JS Buy SDK is in the works. See the [README](https://github.com/Shopify/js-buy-sdk/blob/v1.0alpha/README.md).
```
Comment (swalkinshaw) [inline README.md]: How about:

> This is the `README` for the current stable v0.x version. v1 of JS Buy SDK is under development in the `v1.0alpha` branch. See the [README](https://github.com/Shopify/js-buy-sdk/blob/v1.0alpha/README.md) for more details.

**Comment by jzjiang** [comment]
Updated the note!

File: README.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 # [Shopify](https://www.shopify.com) JavaScript Buy SDK
 [![Circle CI](https://circleci.com/gh/Shopify/js-buy-sdk.png?circle-token=3be0ebe6fbb4841442b86678696947bd4b5456d7)](https://circleci.com/gh/Shopify/js-buy-sdk)
 
+**Note**: This is the `README` for the current stable v0.x version. v1 of JS Buy SDK is under development in the `v1.0alpha` branch.
+See the [README](https://github.com/Shopify/js-buy-sdk/blob/v1.0alpha/README.md) for more details.
+
```
Comment (lshapton) [inline README.md]: v0.x version. v1 ?

Comment (swalkinshaw) [inline README.md]: Just means 0.anything to avoid hardcoding latest minor release number

Comment (jzjiang) [inline README.md]: Does "`v0.x` version. `v1`" look clearer?

Comment (lshapton) [inline README.md]: I think it's fine. 👍

**Pull request merged**